caesars-palace/blocks/stats/stats.css

```diff
@@ -0,0 +1,60 @@
+/*
+Stats Block (2->3->5 columns per row across key media breakpoints)
+*/
+
+div.stats-container {
```

Comment:
I think it's better to scope the CSS to the block, like [here](https://github.com/hlxsites/caesars/blob/main/caesars-palace/blocks/highlight-card/highlight-card.css#L5)

Reply:
Not sure i understand.

![image](https://user-images.githubusercontent.com/1164937/221700268-c3ad666f-951f-4ea2-9f04-629e50a6a2cf.png)